CMakeLists.txt

```diff
@@ -142,22 +151,78 @@ if(SGX)
                             verifier_sgx_ecdsa_qve
                             verifier_sgx_la
                             )
+elseif(WASM)
+    set(DEPEND_LIBS verifier_nullverifier
+                    verifier_sgx_ecdsa
+                    verifier_sev
+                    verifier_sev_snp
+                    verifier_csv
```

Comment:
hi, are you ignore `verifier_tdx_ecdsa`?

Reply:
It's unsupported because of lacking of the TDX PCCS